content/help/faq/distributed-tracing/disabling-tracing.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: How can I prevent Istio from sending trace spans to ZipKin?
-weight: 100
+title: How do I disable tracing?
+weight: 50
```

Comment:
For all renamed or moved pages in this PR, please add an "aliases:" directive for the old paths so that old bookmarks continue working.

See https://preliminary.istio.io/about/contribute/writing-a-new-topic/#renaming-moving-or-deleting-pages